modules/abstract-eth/src/abstractEthLikeCoin.ts

```diff
@@ -135,7 +135,7 @@ export abstract class AbstractEthLikeCoin extends BaseCoin {
     return {};
   }
 
-  isWalletAddress(): boolean {
+  async isWalletAddress(): Promise<boolean> {
```

Comment:
bruh did u mean to change all these???

Reply:
40 files guey

Reply:
why we need to make it async?

Reply:
https://github.com/BitGo/BitGoJS/blob/feafb010da772a8fb454192a44d93bcffcff86a7/modules/sdk-coin-sui/src/sui.ts#L110

isWalletAddress for sui had to be async because of that ^. I had to change the interface to return promise and every coin module implements that interface so that change had to propagate to every coin.